js/components/portal-dashboard/feedback/feedback-question-rows.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Map } from "immutable";
+import Answer from "../../../containers/portal-dashboard/answer";
+import striptags from "striptags";
+import { renderHTML } from "../../../util/render-html";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  answers: any;
+  currentActivity: Map<string, any>;
+  currentStudentId: string | null;
+  feedbacks: Map<any, any>;
+  feedbacksNeedingReview: Map<any, any>;
+  students: Map<any, any>;
+  updateQuestionFeedback: (answerId: string, feedback: any) => void;
+}
+
+export const FeedbackQuestionRows: React.FC<IProps> = (props) => {
+  const { answers, feedbacks, currentActivity, currentStudentId, students, updateQuestionFeedback } = props;
+
+  const handleFeedbackChange = (answerId: string) => (event: React.FormEvent<HTMLTextAreaElement>) => {
+    if (answerId !== undefined) {
```

Comment:
I had a similar comment elsewhere.  `answerId` is typed as a string in the argument list.  If it truly is a string, we shouldn't need this check for undefined.  If it can be undefined, we should type it as such in the argument list: string | undefined or ?string.